src/features/commands/poll.js

```diff
@@ -0,0 +1,145 @@
+                .addStringOption(option =>
+                    option.setName('title')
+                        .setDescription('Title for the poll.')
+                        .setRequired(true))
+                .addBooleanOption(option =>
+                    option.setName('allow-multiple-answers')
+                        .setDescription('Choose whether to allow multiple answers in the poll or not.')
+                        .setRequired(true))
+                .addStringOption(option =>
+                    option.setName('choice-a')
+                        .setDescription('First choice.')
+                        .setRequired(true))
+                .addStringOption(option =>
+                    option.setName('choice-b')
+                        .setDescription('Second choice.')
+                        .setRequired(true))
+                .addStringOption(option =>
+                    option.setName('choice-c').setDescription('Third choice.'))
+                .addStringOption(option =>
+                    option.setName('choice-d').setDescription('4th choice.'))
+                .addStringOption(option =>
+                    option.setName('choice-e').setDescription('5th choice.')))
+        .addSubcommand(subcommand =>
+            subcommand
+                .setName('list')
+                .setDescription('Lists all polls that are created and are active in this server.'))
+        .addSubcommand(subcommand =>
+            subcommand
+                .setName('results')
+                .setDescription('Displays results of a poll.')
+                .addStringOption(option => option.setName("poll-id").setDescription("The poll ID to get results from.").setRequired(true))),
+    async execute(client, interaction) {
+        const pollTitle = interaction.options.getString("title");
+        const pollChoices = [interaction.options.getString("choice-a"), interaction.options.getString("choice-b"), interaction.options.getString("choice-c"), interaction.options.getString("choice-d"), interaction.options.getString("choice-e")].filter((a) => a);
+
+        if (!interaction.guild) {
+            interaction.reply({ content: 'You can only use this command in servers!', ephemeral: true });
+            return;
+        }
+
+        switch (interaction.options._subcommand) {
+            case "create":
+                const multipleAnswersAllowed = interaction.options.getBoolean('allow-multiple-answers', true);
+                strawpoll.createStrawpoll(pollTitle, pollChoices, multipleAnswersAllowed).then(response => {
+                    db.createPoll(interaction.member.user.id, interaction.guild.id, `${pollTitle}:${response.pollId}`).then(resp => {
+                        const publicPollEmbed = new MessageEmbed()
+                            .setColor("#0099ff")
+                            .setTitle(`Poll: ${pollTitle}`)
+                            .setDescription(`You can vote [here](https://strawpoll.com/${response.pollId}).`)
```

Comment:
It would be better to have a button which would lead to that link instead of this.
Description should show options of the poll along with the votes(it would be a new poll so votes would be 0 ofc, but this would make it prettier)

_Also these changes would allow something like dynamic updates to the embed, we could periodically check and update poll embeds in one minute intervals, thus they would look more dynamic. Of course this one would be for future iterations._

Reply:
This one is still pending changes.

Reply:
> > Strange, didn't I change it already?
> 
> No, as you can see in the code and this scr it doesn't have votes defined ![image](https://user-images.githubusercontent.com/11031051/135679529-1e996f12-14d0-48d1-90a5-df18c0618662.png)

Oh